Plugins/DeleteUnusedBranches/DeleteUnusedBranchesForm.cs

```diff
@@ -87,7 +89,7 @@ protected override void OnLoad(EventArgs e)
             BranchesGrid.DataSource = _branches;
         }
 
-        private static IEnumerable<Branch> GetObsoleteBranches(RefreshContext context, string curBranch)
+        private IEnumerable<Branch> GetObsoleteBranches(RefreshContext context, string curBranch)
```

Comment:
Why has this changed to an instance method?

Reply:
Because the actual parsing is now performed by the new GitBranchOutputCommandParser: it's a class field so I remove the static clause on both methods.

I could have instantiated the parser as a local variable, but I prefer to instantiate all the collaborators in the constructor. Anyway, having a consistent code base it's more important so just let me know if you prefer to change it and I'll do my best. ;)

Reply:
I missed that `GetObsoleteBranchNames` references an instance member.
Thank you for the clarification.